Lethal-Lemurs/nodejs-bangazon-site
# Dw auth

## Status
**READY**

## Description
Created the create-products pug template that allows the user to add a product of a logged  in user. The user is also able to designate a product type through a drop down list. The product is then added to the database of Products.

## Resolves Issue Number
#3 and #12 

## Impacted Files in Application
routes/productRoute
controllers/productCtrl
views/create-product.pug
views/partials/nav.pug

## Testing
1. drop your migrations
2. migrate
3. seed
4. Register or Log in
5. In the nav bar click the Sell A Product link
6. Add a product
7. check database for product in the "Products" table
